scheduler/templates/scheduler.html

```diff
@@ -56,10 +56,9 @@ <h2>Schedule for {{ type }} {{ name }}</h2>
                         professor: {{ e.5 }}",
                     backColor: "{{ e.6 }}",
                     barColor: "{{ e.7 }}"
```

Comment:
Wiem, że to nie Twój task ale dodaj proszę  barBackColor: "{{ e.7 }}" to wtedy nie będą się wyświetlać te niebieskie kawałki na tym pasku po lewej.